examples/public/interactivity.html

```diff
@@ -15,6 +15,13 @@
     html, body { margin: 0; padding: 0; }
     #map { position: absolute; top: 0; left: 0; width: 100%; height: 100%; }
   </style>
+  <script>
+    window._trackJs = {
+      token: "cf7a7cd8411140689ea94e92d9fba842",
```

Comment:
Is it secure to expose the trackJs token here?

Reply:
+1 Not sure we can/want to include a trackJs token in the library...

Reply:
It's the way it's shown in their documentation to add the credentials.

And it's there to check if the library was correctly tracking. We can get rid of it.